utils/mil_msgs/action/MoveTo.action

```diff
@@ -8,5 +8,6 @@ float64 linear_tolerance # distance from goal for result to be sent
 float64 angular_tolerance
 ---
 # result
+int32 error # Returns waypoint error
```

Comment:
This would be better as a ```string error``` so it is clear to programs using this what the error means.